platform/ios/src/MGLMapView.h

```diff
@@ -159,7 +159,7 @@ FOUNDATION_EXTERN MGL_EXPORT MGLExceptionName const MGLResourceNotFoundException
  See the <a href="https://www.mapbox.com/ios-sdk/maps/examples/simple-map-view/">
  Simple map view</a> example to learn how to initialize a basic `MGLMapView`.
  */
-MGL_EXPORT IB_DESIGNABLE
```

Comment:
Removing `IB_DESIGNABLE` will stop it from working, but evidently doesn’t remove any code from the shipped framework — are the opportunities here to conditionally remove code (and thus marginally reduce binary size)?